lib/markdown.js

```diff
@@ -16,6 +17,12 @@ class MarkdownLink extends Component {
       );
     }
 
+    if (/^video#/.test(children[0])) {
+      return (
+        <Video src={children[0].split('video#')[1]} />
```

Comment:
`children[0].slice(6)` would be preferred to avoid allocating an array. No change required for merge since I don’t think we’re concerned about memory currently.